multiqc/modules/qualimap/__init__.py

```diff
@@ -1,3 +1,94 @@
+
+
+def parse_numerals(
+    preparsed_d: Dict[str, str],
+    float_metrics: Dict[str, str],
+    int_metrics: Dict[str, str],
+    rate_metrics: Dict[str, str],
+    fpath: str,
+) -> Dict[str, Union[int, float, str]]:
+    """
+    Take pre-parsed Qualimap report (keys to string values), and properly parse
+    numeral values, taking regional formats into account.
+    """
+    # Determine if decimal separator is dot or comma
+    decimalcomma = None
+    for k in rate_metrics:
+        if k in preparsed_d:
+            val = preparsed_d[k]
+            if "," in val and "." in val:
+                log.error(
+                    f"Couldn't determine decimal separator for file {fpath}, as both . and , are "
+                    f"found in a rational value: {val}"
+                )
+                return {}
+            if "," in val:
+                if decimalcomma is False:
+                    log.error(
+                        f"Couldn't determine decimal separator for file {fpath}, as differently formatted"
+                        f"rational values are found"
+                    )
+                    return {}
+                decimalcomma = True
+            if "." in val:
+                if decimalcomma is True:
+                    log.error(
+                        f"Couldn't determine decimal separator for file {fpath}, as differently formatted"
+                        f"rational values are found"
+                    )
+                    return {}
+                decimalcomma = False
+    if decimalcomma is None:
+        # All expected float numbers are integer, so attempt to instead determine the
+        # thousands separator from large int values.
+        for k in int_metrics:
+            if k in preparsed_d:
+                val = preparsed_d[k]
+                if "," in val and "." in val:
+                    log.error(
+                        f"Couldn't determine decimal separator for file {fpath}, as both . and , are "
+                        f"found in a rational value: {val}"
+                    )
+                    return {}
+                if "," in val:
+                    if decimalcomma is True:
+                        log.error(
+                            f"Couldn't determine decimal separator for file {fpath}, as differently formatted"
+                            f"rational values are found"
+                        )
+                        return {}
+                    decimalcomma = False
+                if "." in val:
+                    if decimalcomma is False:
+                        log.error(
+                            f"Couldn't determine decimal separator for file {fpath}, as differently formatted"
+                            f"rational values are found"
+                        )
+                        return {}
+                    decimalcomma = True
+    if decimalcomma is None:
+        log.warning(f"Couldn't determine decimal separator for file {fpath}")
```

Comment:
If we didn't find a decimal then I don't think it really matters, the file should parse fine, right? I could imagine this being relatively common with empty BAM files etc. Suggest we bump down to debug.

```suggestion
        log.debug(f"Couldn't determine decimal separator for file {fpath}")
```